src/material/core/option/option.ts

```diff
@@ -243,6 +233,19 @@ export class _MatOptionBase<T = any> implements FocusableOption, AfterViewChecke
 
 /**
  * Single option inside of a `<mat-select>` element.
+ *
+ * The aria-selected attribute applied to the option conforms to WAI ARIA best practices for listbox
```

Comment:
We shouldn't put the `aria-selected` explanation here, because it's going to show up in the docs site. It can be put right before the `'[attr.aria-selected]': 'selected',` on line 261.